scripts/Dockerfile

```diff
@@ -8,7 +11,22 @@ ADD . $MT_SRC_DIR
 RUN $MT_SRC_DIR/scripts/build.sh
 RUN cp -r $MT_SRC_DIR/build /tmp/build
 
+####################
+# mt-gateway image #
+####################
+FROM alpine:3.10.0 AS mt-gateway
 
+RUN mkdir -p /etc/gw
+COPY scripts/config/storage-schemas.conf /etc/gw/storage-schemas.conf
+COPY --from=gobase /tmp/build/mt-gateway /usr/bin/
+
+EXPOSE 80
```

Comment:
80 is the default, probably makes sense to change that as well

Reply:
why?

Reply:
I suppose I don't have a compelling argument other than using a privileged port feels wrong

Reply:
but this is only inside the docker container, right? if people want to expose this port on their host on a non privileged port, they can do that. that said, we may as well establish a convention here.. since MT is 6060 and this is supposed to sit in front of MT, maybe 6059 ?

Reply:
>since MT is 6060 and this is supposed to sit in front of MT, maybe 6059

Yeah, I was initially thinking of starting at the metrictank port and incrementing by one for any additional ancillary services.  Decrementing works too I suppose